packages/docs/docs/tutorials/bots/bot-basics.md

```diff
@@ -167,15 +167,17 @@ You can find the `id` of your Bot by clicking on the **Details** tab of the Bot
 
 #### `CONTENT_TYPE`
 
-| Content-Type               | typeof `event.input`                         | Description                                                                                                                                                         |
-| -------------------------- | -------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `application/fhir+json`    | [`Resource`](../../api/fhir/resources)       | `<INPUT_DATA>` is parsed as a [FHIR Resource](../../fhir-basics#resources) encoded as a JSON string                                                                 |
-| `text/plain`               | `string`                                     | `<INPUT_DATA>` is parsed as plaintext string                                                                                                                        |
-| `x-application/hl7-v2+er7` | [`HL7Message`](../../sdk/classes/Hl7Message) | `<INPUT_DATA>` is a string that should be parsed as a pipe-delimited HL7v2 message. HL7v2 is a common text-based message protocol used in legacy healthcare systems |
+| Content-Type                         | typeof `event.input`                         | Description                                                                                                                                                         |
+| ------------------------------------ | -------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `text/plain`                         | `string`                                     | `<INPUT_DATA>` is parsed as plaintext string                                                                                                                        |
+| `application/json`                   | `Record<string, any>`                        | `<INPUT_DATA>` is parsed as JSON-encoded object                                                                                                                     |
+| `application/x-www-form-urlencoded ` | `Record<string, string>`                     | `<INPUT_DATA>` is parsed as JSON-encoded object, where all values are treated as strings                                                                            |
```

Comment:
This sentence is ambiguous, maybe slightly incorrect.

"is parsed as a URL-encoded string, resulting in a key/value map" or something.

Reply:
updated